ExtendedRelationOntology.ttl

```diff
@@ -335,7 +335,7 @@ cco:disposition_of_aggregate rdf:type owl:ObjectProperty ;
                              rdfs:subPropertyOf cco:inheres_in_aggregate ;
                              rdfs:domain obo:BFO_0000016 ;
                              rdfs:range obo:BFO_0000027 ;
-                             cco:definition "x disposition_of_aggregate y iff y is an instance of Object Aggregate and x is an instance of Disposition, such that x disposition_of_aggregate y."@en ;
+                             cco:definition "x disposition_of_aggregate y iff y is an instance of Object Aggregate and x is an instance of Disposition, such that x inheres_in_aggregate y."@en ;
```

Comment:
@cameronmore  Please consider changing 'such that' to 'and' throughout this formulations. My read is that the superproperty holds independently of the type instantiation of the relata, but 'such that' implies there is a relationships here.